compiler/rustc_codegen_ssa/src/mir/statement.rs

```diff
@@ -70,6 +70,7 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
                 let op_val = self.codegen_operand(bx, op);
                 bx.assume(op_val.immediate());
             }
+            mir::StatementKind::Intrinsic(box NonDivergingIntrinsic::Expect(..)) => {}
```

Comment:
Could you please also add this at https://github.com/rust-lang/rust/blob/69db514ed9238bb11f5d2c576fe26020e3b99a52/compiler/rustc_codegen_cranelift/src/base.rs#L817?

Reply:
Sure. Normally, only the LLVM backend is build, right? Because I didn't get any compile time errors in GCC or Cranelift

Reply:
`./x.py check` should check all backends. `./x.py build` only builds backends other than LLVM if you tell the build system to with the `rust.codegen-backends` option in `config.toml`.

Reply:
Thanks, I updated this and one more `match` in Cranelift